### PR TITLE
Issue #1339: Handle DelayedBuildChildren error variant

### PR DESCRIPTION
**Pull request by tst-cjeffries**
Add error handling to alleviate a version of the DelayedBuildChildren error.

To test:

1. Build and install OpenGEE (can reliably reproduced on Ubuntu16).
2. Run the historical and tutorial scripts.
3. Verify that the databases are correct, and that `/opt/google/log/gesystemmanager.log` shows that the error was caught for the SFinset_2-5 databases (it should be caught many times).
4. Run the gauge tests (it may be wise to reinstall and clear out the tutorial databases before this step).

### CODE REVIEW

**Comment by tst-lsavoie** [comment]
It looks like this code will work. The only potential problem I see is that you could create extra versions of an asset (you create a version when the build starts, then when DelayedBuildChildren is called you can't load that version so you create a new version). I think we can live with that, though, especially since this is relatively rare.

I didn't do any testing.

File: earth_enterprise/src/fusion/autoingest/AssetGenDaemonCpp.pl [hunk 1]
```diff
@@ -302,38 +302,44 @@ ()
...
-                for (const auto & bi : boundInputs) {
-                    notify(NFY_NOTICE, "             %s", bi.c_str());
-                }
-                notify(NFY_NOTICE, "         version inputs:");
-                for (std::vector<std::string>::const_iterator iv =
-                     version->inputs.begin();
-                     iv != version->inputs.end(); ++iv) {
-                    notify(NFY_NOTICE, "             %s", iv->c_str());
-                }
+                  notify(NFY_NOTICE,
```
Comment (tst-dsugar) [inline earth_enterprise/src/fusion/autoingest/AssetGenDaemonCpp.pl]: besides being a spacing/tab error, do we really want to carry this code block?

Comment (tst-cjeffries) [reply earth_enterprise/src/fusion/autoingest/AssetGenDaemonCpp.pl]: I just further indented the existing formatting. How would you prefer it? I try not to change the existing code much, in hopes of keeping it consistent.

Comment (tst-cjeffries) [reply earth_enterprise/src/fusion/autoingest/AssetGenDaemonCpp.pl]: Cleaned up some of the spacing to reduce the noise.

**Pull request merged**